src/build.ts

```diff
@@ -167,7 +167,7 @@ export async function build(
     const hash = createHash("sha256").update(contents).digest("hex").slice(0, 8);
     const ext = extname(file);
     const alias = `/${join("_file", dirname(file), `${basename(file, ext)}.${hash}${ext}`)}`;
-    aliases.set(loaders.resolveFilePath(file), alias);
+    aliases.set(loaders.resolvePreviewFilePath(file), alias);
```

Comment:
This is confusing because we’re getting the preview path during build, but we’re not really using — we’re using it as the key to the `aliases` map, which we’ll then replace with the hashed value.